.github/workflows/build-haskell.yml

```
@@ -39,6 +39,7 @@ jobs:
           - 9.4.7
           - 9.6.4
           - 9.8.1
```

```suggestion
          - 9.6.5
          - 9.8.2
```

While you're at it, could you bump these compiler versions as well and also update them in the `.cabal` file in the `tested-with` field?

---

Add `9.10.1` as well